src/App.js

```diff
@@ -1,101 +0,0 @@
-import React, { Component } from 'react';
-import './App.css';
-import axios from 'axios'
-
-
-
-const BASE_URL = 'http://localhost:4000/api/animals'
-
-class App extends Component {
-  constructor() {
-    super()
-
-    this.state = {
-      animals: [],
-      newAnimalName: '',
-      newAnimalImage: '',
-      newAnimalDescription: '',
-      animalIdBeingEdited: ''
-    }
-  }
-
-componentDidMount = () => {
-  axios({
-    method: 'GET',
-    url: BASE_URL 
-  }).then(response => {
-    this.setState({ animals: response.data })
-  })
-}
-
-addAnimal = () => {
-  axios({
-    method: 'POST',
-    url: BASE_URL,
-    data: {name: this.state.newAnimalName, imageUrl: this.state.newAnimalImage }
-  }).then(response => {
-    this.setState({ animals: response.data })
-  })
-}
-
-updateDescription = (id) => {
-  axios({
-    method: 'PUT',
-    url: BASE_URL + '/' + this.state.animalIdBeingEdited ,
-    data: { description: this.state.newAnimalDescription }
-  }).then( response => {
-    this.setState({ animals: response.data })
-  })
-}
-
-deleteAnimal = (animalid) => {
-  axios({
-    method: 'DELETE',
-    url: BASE_URL + '/' + animalid
-    
-  }).then( response => {
-    this.setState({ animals: response.data })
-  })
-}
-
-  render() {
-    console.log('this.state.animals: ', this.state.animals)
-    return (
-      <div className="container">
-      <div className='container-row'>
-        <input className='input'
-               placeholder='enter animal name' 
-               onChange={ (event) => this.setState({ newAnimalName: event.target.value }  )} />
-        <input className='input'
-               placeholder='paste image url here' 
-               onChange={ (event) => this.setState({ newAnimalImage: event.target.value }) } />
-        <button className='button' onClick={ () => this.addAnimal() }>save</button>
-      </div>   
```

Comment:
overall I would say to be careful with your indentation - i can show the class how to set up an auto formatter or indenter but it helps a lot